src/PrestaShopBundle/Kernel/ModuleRepository.php

```diff
@@ -67,9 +77,24 @@ public function __construct(Connection $connection, $databasePrefix)
     public function getActiveModules()
     {
         if (null === $this->activeModules) {
-            $sth = $this->connection->query('SELECT name FROM ' . $this->tableName . ' WHERE active = 1');
+            if (null === $this->context || null === $this->context->shop) {
```

Comment:
```suggestion
            if (!isset($this->context->shop)) {
```
Just notice the check, it will be easier :)
Having `$this->context` doesn't mean it has the shop property or the shop property can be private (because we can't trust properties' call :D )